### PR TITLE
Add apply_immediately and performance_insights_enabled vars

### PR DESCRIPTION
**Pull request by andreeapad**

### CODE REVIEW

File: rds.tf [hunk 1]
```diff
@@ -47,25 +47,26 @@ resource "random_id" "snapshot_id" {
...
-  name                      = "beekeeper"
-  username                  = var.db_username
-  password                  = chomp(data.aws_secretsmanager_secret_version.beekeeper_db.secret_string)
-  parameter_group_name      = var.rds_parameter_group_name
-  backup_retention_period   = var.db_backup_retention
-  backup_window             = var.db_backup_window
-  maintenance_window        = var.db_maintenance_window
-  final_snapshot_identifier = "${local.instance_alias}-final-snapshot-${random_id.snapshot_id.hex}"
-  tags                      = var.beekeeper_tags
+  db_name                      = "beekeeper"
```
Comment (javsanbel2) [inline rds.tf:61]: now is db_name?

Comment (andreeapad) [reply rds.tf:61]: yep, the old name is deprecated

File: variables.tf [hunk 2]
```diff
@@ -116,6 +116,18 @@ variable "db_maintenance_window" {
   default     = "wed:03:00-wed:04:00"
 }
 
+variable "db_apply_immediately" {
+  description = "Specifies whether any database modifications are applied immediately, or during the next maintenance window."
+  type        = bool
+  default     = true
```
Comment (abhimanyugupta07) [inline variables.tf:122]: default should be `false` IMO. This is an open source module, others might not want this behaviour.

**Review by javsanbel2 — APPROVED** [review]

File: variables.tf (hunk 2 above)
Comment (andreeapad) [reply variables.tf:122]: ok

Comment (andreeapad) [inline variables.tf:122]:
```suggestion
  default     = false
```

Comment (javsanbel2) [reply variables.tf:122]: I agree on this

**Review by abhimanyugupta07 — APPROVED** [review]

File: variables.tf [hunk 3]
```diff
@@ -116,6 +116,18 @@ variable "db_maintenance_window" {
   default     = "wed:03:00-wed:04:00"
 }
 
+variable "db_apply_immediately" {
```
Comment (abhimanyugupta07) [inline variables.tf:119]: @andreeapad think we missed adding these new variables to the README file.

**Pull request merged**